docs/docker-images/buildkit-standalone.md

```diff
@@ -0,0 +1,61 @@
+```
+
+Omitting the options already discussed from the simple example:
+
+- `-e BUILDKIT_TCP_TRANSPORT_ENABLED=true` makes `buildkitd` listen on a TCP port instead of a Unix socket.
+- `-p 127.0.0.1:8372:8372` forwards the hosts port 8372 to the containers port 8372. When using TCP, `buildkit` will always listen on 8372, but you can configure the apparent port by forwarding a different port on your host.
+
+Assuming you ran this on another machine named `fast-builder`, you could use this remote `buildkitd` by setting `EARTHLY_BUILDKIT_HOST=tcp://fast-builder:8372`, or by specifying the address in your `config.yml`.
+
+## Using This Image
+
+### Requirements
+
+#### Privileged Mode
+
+This image needs to be run as a privileged container. This is because `buildkitd` needs appropriate access to start and run additional containers itself via `runc`.
+
+#### External Usage
+
+To use this image externally, it requires you to forward a port on your machine to the containers port 8372. You will need to ensure that external access ti the machine on the port you chose is possible as well.
+
+If you are using this container locally, please note that `EARTHLY_BUILDKIT_HOST` values where the host is `127.0.0.1`, ` ::1/128`, or `localhost` are considered local and will result in Earthly attempting to manage the buildkit container itself. Consider using your hostname, or another alternative name in these cases.
+
+### Supported Environment Variables
+
+| Variable Name                       | Default Values                 | Description                                                                                                                                                                   |
+|-------------------------------------|--------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| EARTHLY_ADDITIONAL_BUILDKIT_CONFIG  |                                | Additional `buildkitd` config to append to the generated configuration file.                                                                                                  |
+| BUILDKIT_LOCAL_REGISTRY_LISTEN_PORT | `8371`                         | What port should the internal cache registry listen on?                                                                                                                       |
+| BUILDKIT_TCP_TRANSPORT_ENABLED      |                                | Set to `true` when the `buildkitd` instance is going to be used remotely                                                                                                      |
+| BUILDKIT_TLS_ENABLED                |                                | Set to `true` when the `buildkittd` instance will require mTLS from the clients. You will also need to mount certificates into the right place (`/etc/*.pem`).                |
```

Comment:
I wonder if this should be enabled automatically when using TCP so that it is secure by default. So then we would have instead BUILDKIT_TLS_DISABLED or BUILDKIT_TLS_INSECURE.

Reply:
With this image, the code to generate the certs does not exist. Using this standalone is a strictly BYOCerts affair.